src/components/Registration/NatureOfBusiness.vue

```diff
@@ -0,0 +1,224 @@
+<template>
+  <div
+    id="nature-of-business"
+    class="ml-n6 pl-6 rounded-0"
```

Comment:
This is a little confusing. Just curious about the reason here, at first glance you would expect to be relatively the same place without the margin and padding helper classes.

Reply:
Haha, yes, you're right.

The problem this fixes is that NOB is pre-indented by its parent (the reg define business page), so the red validation bar was not at the right place. I put this here (instead of in the parent page) to encapsulate this little idiosyncracy.

Reply:
haha fair enough, all good!